packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.tsx

```diff
@@ -51,20 +59,16 @@ const ContractTypeWidget = ({ is_equal, name, value, list, onChange, languageCha
 
     React.useEffect(() => {
         if (selected_item && selected_item.value !== value) {
-            onChange({ target: { name, value: selected_item.value } });
+            onChange?.({ target: { name, value: selected_item.value } });
```

Comment:
```suggestion
            onChange({ target: { name, value: selected_item.value } });
```
@henry-deriv if onChange is always passed, no need for this :)